maro/simulator/scenarios/supply_chain/topologies/core.yml

```diff
@@ -36,6 +36,9 @@ units:
           # class: class name, joined with the module path to find the class type
           class: "StorageUnit"
           datamodel: "StorageDataModel"
+        SuperStorageUnit:
+          class: "SuperStorageUnit"
+          datamodel:
```

Comment:
Leave it empty or use `None` / an empty string?

Reply:
change to "null"